test/functional/api/cas/cli.py

```diff
@@ -108,6 +108,37 @@ def start_cmd(cache_dev: str, cache_mode: str = None, cache_line_size: str = Non
     return casadm_bin + command
 
 
+def standby_init_cmd(cache_dev: str, cache_id: str, cache_line_size: str,
+                     force: bool = False, shortcut: bool = False):
+    command = " --standby --init "
```

Comment:
cosmetic: remove the space after `--init` (all the params below have a preceding space)